packages/api-explorer/src/index.jsx

```diff
@@ -209,6 +209,11 @@ class ApiExplorer extends React.Component {
     const NewGlossaryTermsContext = utils.GlossaryContext;
     /* eslint-enable global-require */
 
+    const onAuthGroupChange = auth => {
```

Comment:
Since you're calling a class method `onAuthGroupChange` of the same name, for consistency and maintainability it might make sense to consolidate prop call inside `this.onAuthGroupChange()`. 

On 117:
```
 if (this.props && this.props.onAuthGroupChange) return this.props.onAuthGroupChange(group);
```